<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightDialog</class>
 <widget class="QDialog" name="LightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oświetlenie</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonLight0">
     <property name="text">
      <string>Światło dzienne (lekkie)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonLight1">
     <property name="text">
      <string>Światło dzienne (mocne)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonLight2">
     <property name="text">
      <string>Światło białe rozproszone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonLight3">
     <property name="text">
      <string>Światło żółte skierowane</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxLight">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
